### ISSUE TITLE
Feature request: Allow configuration of php code lens file pattern

### ISSUE THREAD

**Issue by BladeMF** [elonmallin/vscode-phpunit#103]
It will be useful to be able to configure where code lens is enabled in php files. Not all folders with tests start with the text `test`. They may _end_ with `tests` as well or have some other naming rule. 

I can PR this.

**Comment by elonmallin** [comment]
Hi, my understanding is that phpunit naming convention requires test methods to start with `test*` or that the test attribute in a comment is used to be identified as a test.

If that's the case then I dont think we should add custom naming rules that arent officially supported in phpunit.

I found this in the docs here https://docs.phpunit.de/en/10.5/writing-tests-for-phpunit.html

**Comment by BladeMF** [comment]
You are correct, but the filter I am talking about is the _directory_ filter, not the method filter. Code lens is not activated unless the directory name _starts_ with `test`, which should be configurable.

**Comment by elonmallin** [comment]
Ah I see, please make a PR if you want :)
